apps/worker/tasks/test_analytics_notifier.py

```diff
@@ -130,6 +130,8 @@ def run_impl(
                     attempted=False,
                     succeeded=False,
                 )
+            except LockRetry as retry:
```

Comment:
it's more common for exceptions to be `as e`, given that there is already a `retry` method, I think we should change this to a different var name